internal/host/validator.go

```diff
@@ -387,17 +387,14 @@ func (v *validator) printDiskEncryptionRequirementsSatisfied(c *validationContex
 	case ValidationSuccess:
 		return fmt.Sprintf("Installation disk can be encrypted using %s", *c.cluster.DiskEncryption.Mode)
 	case ValidationFailure:
-		if c.host.Role == models.HostRoleAutoAssign && !isAutoAassignAllowed(c) {
-			return "Host role auto assignment is supported only if encryption is enabled on all nodes or disabled"
-		}
 		if c.inventory.TpmVersion == models.InventoryTpmVersionNone {
 			return "TPM version could not be found, make sure TPM is enalbed in host's BIOS"
 		} else {
 			return fmt.Sprintf("The host's TPM version is not supported, expected-version: %s, actual-version: %s",
 				models.InventoryTpmVersionNr20, c.inventory.TpmVersion)
 		}
 	case ValidationPending:
-		return "Missing host inventory"
+		return "Missing host inventory or role assignment"
```

Comment:
Please split the return value to be specific for each case.

Reply:
Done